app/src/components/Modals/add-user-modal.tsx

```diff
@@ -140,7 +141,7 @@ const AddUserModal: FC<AddUserModalProps> = ({
     <>
       <Modal blockScrollOnMount={false} isOpen={isOpen} onClose={onClose}>
         <ModalOverlay />
-        <ModalContent minH="600px" minW="568px" marginTop="10%">
+        <ModalContent minH="300px" minW="568px" marginTop="10%">
```

Comment:
Why is this a percentage? This means that the margin will be different depending on the page/ screen size?